addons/payment/static/src/scss/payment_form.scss

```diff
@@ -33,6 +33,11 @@
     .panel-body {
         border-top: 1px solid #ddd;
         padding: 1.14em !important;
+
+        .padded p {
+            margin-left: 23px;
+        }
```

Comment:
Seems like this modification should be isolated in a single commit with a clear description of what it improves.